cfgov/jinja2/v1/_includes/molecules/call-to-action.html

```diff
@@ -21,9 +21,8 @@
    settings.button.text:    The text to display on button.
 
    ========================================================================== #}
-
-<div class="m-call-to-action">
-    {% if value %}
+{% if value %}
```

Comment:
I know this is temp code, but still seems like this would be easier:
```
{% if value %}
    {{ render(value) }}
{% endif %}
{% macro render(settings) %}
    …
{% endmacro %}
```

@kave Any reason that wouldn't work?